src/Streamly/Internal/Data/Fold.hs

```diff
@@ -193,10 +193,6 @@ module Streamly.Internal.Data.Fold
 
     -- * Unzipping
     , unzip
-    -- These can be expressed using lmap/lmapM and unzip
```

Comment:
This is an important fact to remember. Or better we can have an unzip based definition of these in the haddock docs just to remind the user the relation between unzip and these.